test/rack/session/redis_test.rb

```diff
@@ -40,60 +40,60 @@
   it "can create it's own pool" do
     session_store = Rack::Session::Redis.new(incrementor, pool_size: 5, pool_timeout: 10)
     conn = session_store.instance_variable_get(:@conn)
-    conn.pool.class.must_equal ::ConnectionPool
-    conn.pool.instance_variable_get(:@size).must_equal 5
+    _(conn.pool.class).must_equal ::ConnectionPool
```

Comment:
Why did you surround all the expectations with this? I understand that a future Minitest version will require it, but if it wasn't blocking you, wrapping the expectations for every test not necessary for this change. We'll do a full overhaul of all redis-store repos that use this syntax when we need to in the future.

Reply:
@tubbo sorry, it was little annoying while I was testing changes, so I decided to remove this deprecations. I can reset them, if you want.

Reply:
@tubbo should I rollback this tests changes?

Reply:
yeah I think we should lock down Minitest to an earlier version before it started generating these warnings. in redis-rack-cache's gemspec, I did the following:

```ruby
  s.add_development_dependency 'minitest', '~> 5.11.0'
```

that will make the diff a lot easier to read. in the future i'll change this everywhere and depend on the right version in redis-store-testing

Reply:
@tubbo ok, one moment